dev-vcs/topgit/topgit-0.19.12.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+DESCRIPTION="A different patch queue manager"
+HOMEPAGE="https://mackyle.github.io/topgit/topgit.html https://github.com/mackyle/topgit"
+SRC_URI="https://github.com/mackyle/${PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-apps/sed
+	virtual/awk"
```

Comment:
How are these required in DEPEND? Maybe BDEPEND but that still seems weird. 

Also I believe both are already provided by `@system` set.

Reply:
Yes, that correct, need to BDEPEND. Just leave them as explicit dependency, maybe there system don't have sed and awk?